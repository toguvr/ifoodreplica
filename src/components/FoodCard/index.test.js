import React from 'react'
import renderer from 'react-test-renderer';
import { StyledCard, FlexDiv, StyledCardMedia, CardContent, 
    TypographyTitle,  TypographyDescription, TypographyValue, 
    TypographyButtonText, AddRequestButton } 
from './styled';

describe("AddRequestButton", () => {
    it("Should match Snapshot", () => { 
        const tree = renderer
        .create(<AddRequestButton />).toJSON();
        expect(tree).toMatchSnapshot()
    })
})

describe("StyledCard", () => {
    it("Should match Snapshot", () => {
        const tree = renderer
        .create(<StyledCard />).toJSON();
        expect(tree).toMatchSnapshot()
    })
})

describe("StyledCardMedia", () => {
    it("Should match Snapshot", () => {
        const tree = renderer
        .create(<StyledCardMedia />).toJSON();
        expect(tree).toMatchSnapshot()
    })
})

describe("CardContent", () => {
    it("Should match Snapshot", () => {
        const tree = renderer
        .create(<CardContent />).toJSON();
        expect(tree).toMatchSnapshot()
    })
})

describe("FlexDiv", () => {
    it("Should match Snapshot", () => {
        const tree = renderer
        .create(<FlexDiv />).toJSON();
        expect(tree).toMatchSnapshot()
    })
})

describe("TypographyTitle", () => {
    it("Should match Snapshot", () => {
        const tree = renderer
        .create(<TypographyTitle />).toJSON();
        expect(tree).toMatchSnapshot()
    })
})

describe("TypographyDescription", () => {
    it("Should match Snapshot", () => {
        const tree = renderer
        .create(<TypographyDescription />).toJSON();
        expect(tree).toMatchSnapshot()
    })
})

describe("TypographyValue", () => {
    it("Should match Snapshot", () => {
        const tree = renderer
        .create(<TypographyValue />).toJSON();
        expect(tree).toMatchSnapshot()
    })
})

describe("TypographyButtonText", () => {
    it("Should match Snapshot", () => {
        const tree = renderer
        .create(<TypographyButtonText />).toJSON();
        expect(tree).toMatchSnapshot()
    })
})