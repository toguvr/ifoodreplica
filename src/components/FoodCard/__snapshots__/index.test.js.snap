// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddRequestButton Should match Snapshot 1`] = `
<div
  className="sc-dnqmqq fbSqyp"
/>
`;

exports[`CardContent Should match Snapshot 1`] = `
<div
  className="sc-bxivhb pmVMD"
/>
`;

exports[`FlexDiv Should match Snapshot 1`] = `
<div
  className="sc-bwzfXH jlrUAl"
/>
`;

exports[`Request Should match Snapshot 1`] = `
<div
  className="sc-dnqmqq fbSqyp"
/>
`;

exports[`StyledCard Should match Snapshot 1`] = `
<div
  className="sc-bdVaJa dEQyYW"
/>
`;

exports[`StyledCardMedia Should match Snapshot 1`] = `
<div
  className="MuiCardMedia-root-1 sc-htpNat cXoTGc"
/>
`;

exports[`TypographyButtonText Should match Snapshot 1`] = `
<span
  className="sc-htoDjs bCQbSu"
/>
`;

exports[`TypographyDescription Should match Snapshot 1`] = `
<span
  className="sc-bZQynM MfOWc"
/>
`;

exports[`TypographyTitle Should match Snapshot 1`] = `
<span
  className="sc-ifAKCX kxofPk"
/>
`;

exports[`TypographyValue Should match Snapshot 1`] = `
<span
  className="sc-gzVnrw gdkpaK"
/>
`;
